wls-admin-service/src/test/java/de/muenchen/oss/wahllokalsystem/adminservice/configuration/SecurityConfigurationTest.java

```diff
@@ -194,4 +194,30 @@ void should_returnOk_when_callingAuthenticated() throws Exception {
         }
     }
 
+    @Nested
+    class Deletetermindaten {
```

Comment:
warum heißt es einmal termindaten und einmal wahltermindaten?
```suggestion
    class DeleteWahltermindaten {
```